### PR TITLE
Enable linters for more directories

### PR DESCRIPTION
**Pull request by VenelinMartinov**
This enables the golangci-lint linters for more directories, notably `pkg/pf` which wasn't linted before

fixes https://github.com/pulumi/pulumi-terraform-bridge/issues/2474

### CODE REVIEW

**Review by iwahbe — APPROVED** [review]
Looks great. I'm assuming that the changes are automated, and I don't need to individually review them.

File: pkg/pf/internal/check/not_supported.go [hunk 1]
```diff
@@ -83,7 +83,7 @@ type notSupportedUtil struct {
 	sink diag.Sink
 }
 
-func (u *notSupportedUtil) warn(format string, arg ...interface{}) {
+func (u *notSupportedUtil) warnf(format string, arg ...interface{}) {
```
Comment (VenelinMartinov) [inline pkg/pf/internal/check/not_supported.go:86]: This bumped a linter warning that it should be renamed.

**Review by VenelinMartinov — COMMENTED** [review]
The non-automatic changes were:

Renaming the alias for importing `pulumirpc` - it is now standardized as `pulumirpc`
Splitting long lines

and the two marked with comments

**Comment by VenelinMartinov** [comment]
@iwahbe the only real change is marked above and it seems like a good catch

**Pull request merged**